experiments/download_experts.py

```diff
@@ -0,0 +1,14 @@
+import urllib
```

Comment:
This is OK for now, but I feel like making an S3 bucket or GCP Cloud Storage will be cleaner in the long run. If we want to add or update a model, we have to make a new ZIP file, change the URL, etc.

It's pretty easy to sync with S3, see e.g. https://github.com/HumanCompatibleAI/adversarial-policies/blob/master/experiments/pull_public_s3.sh